### DISCUSSION TITLE
第1节课笔记(24班)

### DISCUSSION THREAD

**Discussion by zhjunqin** [InternLM/Tutorial#203] [笔记]
第一节课笔记（24班）

**Comment by waurblove** [comment]
https://www.bilibili.com/read/cv29305256/?spm_id_from=333.999.0.0&jump_opus=1

**Comment by yezhou05** [comment]
https://app.yinxiang.com/fx/1b9524c5-8adf-4f6b-b368-ef3e32620370

**Comment by Flora-Chai** [comment]
【(1)书生·浦语大模型全链路开源体系-哔哩哔哩】 https://b23.tv/mSvHxUF

**Comment by PopoData** [comment]
https://zhuanlan.zhihu.com/p/676950139

**Comment by AnyaWoo** [comment]
https://blog.csdn.net/weixin_58810992/article/details/135499477

**Comment by ljn12yyds** [comment]
https://blog.csdn.net/ljn20001229/article/details/135508500?spm=1001.2014.3001.5502

**Comment by williamleo0730** [comment]
https://zhuanlan.zhihu.com/p/677117655?

**Comment by ch521-od** [comment]
https://blog.csdn.net/qq_74261455/article/details/135522544?spm=1001.2014.3001.5502

**Comment by lindsey-chang** [comment]
[https://blog.csdn.net/qq_37397652/article/details/135532014](https://blog.csdn.net/qq_37397652/article/details/135532014)

**Comment by kelenlv** [comment]
https://github.com/kelenlv/LLM_notes/blob/main/Day1.md

**Comment by aaashinshin** [comment]
https://blog.csdn.net/weixin_44254889/article/details/135556462

**Comment by LibraMATT** [comment]
https://github.com/LibraMATT/Puyu-LLM-camp-notes/blob/main/course1.md

**Comment by yansuoyuli** [comment]
第一节课笔记：https://blog.csdn.net/m0_58431543/article/details/135567973

**Comment by flh1107** [comment]
第一课笔记：https://www.bilibili.com/read/cv29515009/

**Comment by Xinyi-Cheng** [comment]
第1节课笔记：https://zhuanlan.zhihu.com/p/677639566

**Comment by half-lang** [comment]
https://github.com/half-lang/Intern-homework/blob/main/%E4%B9%A6%E7%94%9F%E6%B5%A6%E8%AF%AD%E4%BD%9C%E4%B8%9A1.ipynb

**Comment by JacobLiu-S** [comment]
https://www.bilibili.com/read/cv29518456?spm_id_from=333.788.0.0 课程一

**Comment by robotzm** [comment]
https://blog.csdn.net/weixin_48319544/article/details/135570918?spm=1001.2014.3001.5502
